File: pascals-triangle-ii/pascals-triangle-ii.py
class Solution:
    def getRow(self, rowIndex: int) -> List[int]:
        ans=[1]
        for i in range(rowIndex):
            aux=[1]
            for i in range(len(ans)-1):
                aux.append(ans[i]+ans[i+1])
            aux.append(1)
            ans=aux
        return ans
                

File: pascals-triangle-ii/pascals-triangle-ii.java
class Solution {
    public List<Integer> getRow(int rowIndex) {
        List<Integer> ans= new ArrayList<Integer>();
        ans.add(1);
        for(int i=0;i<rowIndex;i++){
            List<Integer> aux=new ArrayList<Integer>();
            aux.add(1);
            for(int x=0;x<i;x++){
                aux.add(ans.get(x)+ans.get(x+1));
            }
            aux.add(1);
            ans=aux;
            
        }
        return ans;
    }
}